full translation of c00 (remaining c00/30/21->32, c00/34, c00/50) (#72)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/50/50/0/translation.txt

// Dumped for Honey Select 2 v1.2.0 by Text Dump v1.4.3
//
//attitude:favor

> この様な雰囲気のある場所であなたとお酒を飲むとなると、この後のことを期待してしまいます。=Having drinks with you in a place with such an atmosphere makes me look forward to what comes after.

Reviewer: ```suggestion
この様な雰囲気のある場所であなたとお酒を飲むとなると、この後のことを期待してしまいます。=Drinking with you in a place like this, I'm looking forward to what comes next.
```

初めからそのつもりで誘って下さったのですよね？　とぼけても無駄ですよ？=You asked me out with the intention of doing this from the beginning, didn't you? There's no point in playing dumb, you know?
本日はどの様なセックスをして下さるのですか？　期待しています。=What kind of sex are we going to have today? I'm looking forward to it.
酔いもまわってきたところで、セックス致しませんか？=Now that you've sobered up, shall we have sex?
そうですね……火照った身体を冷やすためにも、プールでするのが宜しいかと。=Let's see... If you want to cool off, I think it would be best to do it the pool.
おや、想像されたのですね？　もう固く膨らんでいますよ。では、準備も整ったところで始めましょう。=Oh my, it looks like you've been imagining it all along. You're already bulging hard. Since you're ready, let's get started.
ムードのある場所でのんびりするのも好きですが、そろそろ情熱的な愛を頂きたいです。=I wouldn't mind just relaxing here, but it's time we made some pasionate love.

> ……直接、口に出した方が興奮するのでしたら……今すぐ私を抱いて頂けませんか？　身体があなたを求めているのです。=...Since it excites you when I say I'm more direct... Can you take me right now? My body needs you.

Reviewer: Just had a thought. The opening `...` implies she's waiting for him, but he didn't say anything. We could keep that feeling with something like (otherwise fine as is)

```suggestion
……直接、口に出した方が興奮するのでしたら……今すぐ私を抱いて頂けませんか？　身体があなたを求めているのです。=Oh, you want me to beg... Can you take me right now? My body needs you.
```

Author: yeah, it's a nifty idea, but i feel the initial suggestion fits better, somehow. or,a combination of the two: "...Since you want me to beg... Can you take me right now? My body needs you." well, i still give the edge in accuracy to your first suggestion, so i'll keep it for now, but if you want one of these, let me know and i'll add them there, they are good as well.

> ……直接、口に出した方が興奮するのでしたら……今すぐ私を抱いて頂けませんか？　身体があなたを求めているのです。=...If it's exciting for you when I say I'm more explicit... Could you take me right now? My body wants you.

Reviewer: ```suggestion
……直接、口に出した方が興奮するのでしたら……今すぐ私を抱いて頂けませんか？　身体があなたを求めているのです。=....Since it excites you more when I'm direct... Can you take me right now? My body needs you.
```

Reviewer: Continued [here](https://github.com/IllusionMods/HS2-Translation/pull/72/files#r673037390) (wait for it)

> お酒の次は、あなたに酔わせて下さい。はしたない女の願いを叶えて頂けますか？=After alcohol, please let me get drunk on you. Will you grant the wish of this shameful woman?

Reviewer: ```suggestion
お酒の次は、あなたに酔わせて下さい。はしたない女の願いを叶えて頂けますか？=After the alcohol, please let me get drunk on you. Will you grant the wish of a slut like me?
```
